.github/workflows/clean_pr.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           base="$(git merge-base "origin/$GITHUB_BASE_REF" 'HEAD^2')"
           am="$(git log "$base..HEAD^2" --pretty=tformat: --name-status --diff-filter=AM |
                 cut --fields 2 | sort | uniq --repeated |
-                grep -E '.(png|pdf|ps|eps|svg)' || true)"
+                grep '\-E .(png|pdf|ps|eps|svg)' || true)"
```

Comment:
Can you please explain this change?

Reply:
Yes, apologies in advance because I made a mistake here. With the original line, the regex `grep -E '.(png|pdf|ps|eps|svg)' || true)"` makes the pr_clean action fails, as reported on gitter. The grep looks for images, but in  `tutorials/text/text_props.py` some words are wrongly identified. 
For example, a quick `cat tutorials/text/text_props.py | grep -E '.(png|pdf|ps|eps|svg)'` returns:

```
variant                     [ ``'normal'`` | ``'small-caps'`` ]
# | ``'font.variant'``  | Default variant, ex ``'normal'``, ``'small-caps'`` |
# </tutorials/text/text_props>`.
# On linux, `fc-list <https://linux.die.net/man/1/fc-list>`__ can be a
```
I think the correct regex expression to identify images should be `grep -E '\.(png|pdf|ps|eps|svg)' || true)"`.

Reply:
Ah, got it. As a hint: It's usually a good idea to make separate PRs for unrelated topics. The combination can cause confusion and makes PRs harder to review. Also, it would not be possible to search through the PR history and find that fix. Lastly, the small fix would be easier to review an can get in faster than a new feature.

Anyway, since this PR is basically done, I'll leave it up to you if you want to split out that fix.